client/components/QuestionsAndAnswers/AnswerItem.jsx

```diff
@@ -18,34 +18,33 @@ const AnswerItem = ({ answer }) => (
         {' '}
       </AnswererName>
       <AnswerDate>
-        {answer.date}
+        {new Date(answer.date).toLocaleDateString('en-US', {year: 'numeric', month: 'long', day: 'numeric'})}
       </AnswerDate>
       <AnswerHelpfulness>
         Helpful?
         {'   '}
       </AnswerHelpfulness>
-      <UnderLineYes>
+      <UnderLineYes onClick={() => console.log('is this answer helpful?')}>
         {'   '}
         Yes
         {' '}
       </UnderLineYes>
       <AnswerHelpfulness>
-        (
-        {answer.helpfulness}
-        )
+        ({answer.helpfulness})
       </AnswerHelpfulness>
       <Divide>
         |
       </Divide>
-      <Report>
+      <Report onClick={() => console.log('report this answer?')}>
         Report
       </Report>
     </AnswerDetails>
   </AnswerContainer>
 );
 
 AnswerItem.propTypes = {
-  answer: PropTypes.isRequired,
+  answer: PropTypes.any.isRequired
+  // answer: PropTypes.arrayOf(PropTypes.object).isRequired
 };
 
 // style
```

Comment:
It might be easier to make the children of answer details to be spans instead of h6. I think they might all fit in one line easier that way without too much manipulation.